book.json

```diff
@@ -1,6 +1,6 @@
 {
     "plugins": [
-        "theme-bandwidth@1.9.11",
+        "theme-bandwidth@1.9.13",
```

Comment:
Should it be .12? Based on https://github.com/Bandwidth/docs-theme/pull/58/files

Reply:
had a 🐛 it got bumped to `.13` :(

Reply:
cool